javascript/javascript3/week1/currencyCnvrtrWeek1.html

```diff
@@ -0,0 +1,163 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Currency Converter Form</title>
+    <link rel="stylesheet" href="currencyCnvrtrWeek1.css" />
```

Comment:
This file is not in the current folder so it throws an error in the browser console.

Reply:
Hi vdaguenet, 

I forgot to push css file before. Now I pushed it. Please hava a look into it.

With Regards,
Rajesh Bhatt